code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -188,7 +188,7 @@ var/global/list/damage_icon_parts = list()
 
 	var/husk = (M_HUSK in src.mutations)
 	var/fat = (M_FAT in src.mutations) && (species && species.anatomy_flags & CAN_BE_FAT)
-	var/hulk = (M_HULK in src.mutations) && !ishorrorform(src) && mind.special_role != HIGHLANDER // Part of the species.
+	var/hulk = (M_HULK in src.mutations) && !ishorrorform(src) // Part of the species.
```

Comment:
?

Reply:
null.mind is a problem, also if you're highlander and die you lose highlander, I think that's how it should work

Reply:
y'know what maybe best I dont touch it